datatypes/src/raster/helpers.rs

```diff
@@ -192,7 +192,7 @@ macro_rules! call_bi_generic_raster2d_staircase {
     };
 }
 
-/// Generates a a `TypedRaster2D` by calling a function for all variants.
+/// Generates a a `TypedRaster2D` by calling a function on its variant.
```

Comment:
on whose variant? In contrast to the other macros this one doesn't match a TypedRaster2D variant but a RasterDataType variant

Reply:
Just make a suggestion, you are right in principle.

Reply:
```suggestion
/// Generates a a `TypedRaster2D` by calling a function with the specified data type variant.
```